drivers/mpsl/clock_control/nrfx_clock_mpsl.c

```diff
@@ -3,28 +3,53 @@
  *
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
-
+#include <nrf.h>
```

Comment:
This will be included through `<nrfx_clock.h>`. But anyway, do you actually need this inclusion?

Reply:
I added it because something was failing at some point. Personally I prefer to include required headers explicitly. If you insist I can remove it.

Reply:
I've removed it to speed up the PR.